stock_picking_move_line_descriptions/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Adds Description column of Purchase in Picking
```

Comment:
Suggested (description is for move lines; purchase order lines is only one of the possible origins):
```suggestion
Adds Description column to Picking Detailed Operations
```
The same in `__manifest__` file